admin/app/components/solidus_admin/ui/table/component.html.erb

```diff
@@ -9,6 +9,12 @@
   "
   data-controller="<%= stimulus_id %>"
 >
+  <% toolbar_classes = "h-14 p-2 bg-white border-b border-gray-100 justify-start items-center gap-2 flex" %>
+
+  <div class="<%= toolbar_classes %>">
```

Comment:
Nit: What about inlining the classes? They're defined just a couple of lines above :slightly_smiling_face:

Reply:
It's going to be reused in the batch actions, I could have inlined them now and extracted them back later, but since I took the commit out of that branch it came up like this.

I would skip the inlining knowing it's gonna change again soon. It's also possible that the toolbar will become its own component sooner or later.

Reply:
@waiting-for-dev nvm, just inlined, I had to rebase anyway, thanks for suggesting it! 🙏